frontend-plugin-core/src/main/java/com/github/eirslett/maven/plugins/frontend/lib/version/manager/VersionManagerCache.java

```diff
@@ -0,0 +1,48 @@
+package com.github.eirslett.maven.plugins.frontend.lib.version.manager;
+
+import java.io.File;
+
+public class VersionManagerCache {
+
+    private VersionManagerType versionManagerType;
+
+    private File nodeExecutable;
+    private File npmExecutable;
+
+    public VersionManagerCache(VersionManagerType versionManagerType) {
+        this.versionManagerType = versionManagerType;
+    }
+
+    public VersionManagerCache() {
+
+    }
+
+    public VersionManagerType getVersionManagerType() {
```

Comment:
super nit: don't really gain anything from getters/setters when this isn't API. IDEs ca refeactor this almost instantly.

Reply:
Yup, it was autogenerated by IDE in the first place. Some of the getters have extra logic so using getters for everything makes it uniform